modules/client-configuration/pages/client-proxy.adoc

```diff
@@ -21,4 +22,69 @@ You can see a list of clients that are connected to a proxy by clicking the name
 
 A list of chained proxies for a Salt client can be seen by clicking the name of the client in menu:Systems[All], then select the [guimenu]``Connection`` subtab of the [guimenu]``Details`` tab.
 
-If you decide to move any of your clients between proxies or the server you need to repeat the registration process from the beginning.
+
+
+== Move Clients between Proxies
+
+You can move Salt and Salt SSH push clients between proxies without the need to repeat the registration process.
+
+
+
+[NOTE]
+====
+If you want to move a traditional client between proxies or any client between chained proxies you must repeat the registration process from the beginning.
+====
```

Comment:
This is not correct.
Moving salt client between chained proxies is supported.
Moving chained proxy server is not supported even if it is managed by salt. Proxy server must always registered from the beginning.

Reply:
ok, thanks.  I got it wrong.  I think we can simplify it.